YidaHu_chat-knows_api_101.py

```
@@ -0,0 +1,57 @@
+model_name = embedding_model_dict['text2vec-base']
+model_kwargs = {'device': 'cpu'}
+embeddings = HuggingFaceEmbeddings(model_name=model_name,
+                                   model_kwargs=model_kwargs)
+loader = UnstructuredTxtLoader()
+app = FastAPI()
+
+doc_search = None
+
+
+@app.post("/uploadfile/")
+async def upload_file(file: UploadFile = File(...)):
```

Ruff (0.13.1)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)